client/blocks/reader-full-post/_content.scss

```diff
@@ -286,7 +286,7 @@
 	color: #fff;
 }
 .wp-block-social-links:not(.is-style-logos-only) .wp-social-link-youtube {
-	background-color: red;
+	background-color: #f00;
```

Comment:
Just to note, `red / #f00` is the official YouTube red so not changing to variable here.